gradle.properties

```diff
@@ -5,7 +5,7 @@ org.gradle.parallel.threads=4
 org.gradle.jvmargs=-Xmx2G
 
 modGroup=club.sk1er
-modVersion=1.0
+modVersion=0.1
```

Comment:
Was this intentional?

Reply:
im fine with this one personally, would alert users when 1.0 is officially released